client/src/unifyfs-dirops.h

```diff
@@ -1,8 +1,8 @@
 /*
- * Copyright (c) 2017, Lawrence Livermore National Security, LLC.
```

Comment:
In some commercial software I used to work on, and with ZFS code, the rule was to add the year to the copyright line (like `Copyright (c) 2017, 2020`) rather than update the year.  I'm not a lawyer, and I have no idea if that's what actually should be done, or even if it's relevant here, just wanted to mention it.

Reply:
Thanks, I've addressed this.